.github/workflows/milestone.yml

```diff
@@ -0,0 +1,83 @@
+name: milestone
+
+on:
+  push:
+    branches: [master]
```

Comment:
We don't use master branch in this repository.

Reply:
dang .. good spotting

Reply:
Updated to main